db/seeds.rb

```diff
@@ -2,7 +2,7 @@
 
 if Rails.env.development?
   token = AuthToken.create(user: user, user_agent: 'Rails')
-  token.update(hashed_secret: BCrypt::Password.create('secret', cost: 1), device_id: 'device-id')
+  token.update(device_id: 'device-id')
 end
```

Comment:
Does it make sense to still add this? IIUC there is no way to use this token right?

Reply:
It's pretty easy to use this token by getting the following in a rails console: `AuthToken.first.generate_token_for(:api)`. I tend to use this when trying things in the api, so that I can make request using curl rather having to go through the web client.
(I mean, it isn't much more work to do something like `FactoryBot.create(:auth_token, user: User.first).generate_token_for(:api)`)